### DISCUSSION TITLE
IINA 1.3.4

### DISCUSSION THREAD

**Discussion by lhc70000** [iina/iina#4738] [Announcements]
## New
- Added a new "Take Screenshot" button on the OSC https://github.com/iina/iina/pull/4648

## Bug fixes
- Fixed multiple localization issues https://github.com/iina/iina/issues/4644 https://github.com/iina/iina/issues/4610 https://github.com/iina/iina/pull/4732, text clipping issue in France https://github.com/iina/iina/pull/4735
- Fixed duplicate plugin not detected when trying to install https://github.com/iina/iina/issues/4410
- Fixed more unexpected quits https://github.com/iina/iina/issues/4725
- Fixed sometimes black windows doesn't disappear https://github.com/iina/iina/issues/4683

## Improvements
- Download from Open Subtitles using the new REST API https://github.com/iina/iina/issues/3920
- Improved the experience for Assrt https://github.com/iina/iina/issues/4671
- The menu entry "Music Mode" now becomes "Enter Music Mode"/"Exit Music Mode" depending on the current mode, localization will arrive in the next release https://github.com/iina/iina/issues/4463

## Plugin related updates

Please note that the plugin system is still under development, and we aim to make it stable in the next release. To try out the new plugin system, please run `defaults write com.colliderli.iina iinaEnablePluginSystem -bool YES` in your Terminal. The documentation is available at [https://docs.iina.io/](https://docs.iina.io/index.html).

- Add `iina-plugin` helper tool  https://github.com/iina/iina/pull/4442
- Plugin: implement WebSocket API for macOS 10.15 or later https://github.com/iina/iina/pull/4586
- Plugin: add `input` module https://github.com/iina/iina/pull/4569

----

#### SHA256

```
7de50f5ad4a2fd5b27bf59a31b2060074c05331c7aafa533ad4adab65028d78c
```

<hr /><em>This discussion was created from the release <a href='https://github.com/iina/iina/releases/tag/'>IINA 1.3.4</a>.</em>

**Comment by valynor** [comment]
Same as last release: it's now 17 hours post release and the updater still tells me 1.3.3 is the newest version available ..

**Reply by lhc70000** [reply]
The auto-update file has just been published, and you should be able to find an update soon! After uploading the DMG to GitHub, we need to wait for it to sync to our mirror, which can take up to 12 hours. Thank you for the reminder, and Happy New Year!
